sdcm/results_analyze/__init__.py

```diff
@@ -188,6 +188,7 @@ def check_regression(self, test_id, data, is_gce=False):  # pylint: disable=too-
         last_events, events_summary = self.get_events(event_severity=[
             Severity.CRITICAL, Severity.ERROR, Severity.WARNING])
         reactor_stall_events, reactor_stall_events_summary = self.get_debug_events()
+        self.log.info(f'FFFFFF reactor_stall_events_summary={reactor_stall_events_summary} FFFFFF')
```

Comment:
here i need to remove this debugging message, and in the previous line, i might want to filter out only the reactor stall events (from within debug events)